src/components/button/_button.scss

```diff
@@ -176,7 +176,8 @@
     letter-spacing: 0;
   }
 
-  .#{$prefix}--btn--secondary + .#{$prefix}--btn--primary {
+  .#{$prefix}--btn--secondary + .#{$prefix}--btn--primary,
+  .#{$prefix}--btn--tertiary + .#{$prefix}--btn--danger--primary {
```

Comment:
Wondering if this line should simply be `.#{$prefix}--btn + .#{$prefix}—btn`, probably in next major version? CC @carbon-design-system/ui

Reply:
I think the intention here was that there should never be more than one primary button? I'm not completely sure though.

Reply:
@alisonjoseph @asudoh ☝️ I'm pretty sure that's the reasoning when @hellobrian was working on this.

Reply:
I think so too 👍